src/cms/data/park-urls.json

```diff
@@ -1,1036 +1 @@
{"orcs":"230","url":"https://bcparks.ca/parks/stuart-lake/","oldUrl":"https://bcparks.ca/explore/parkpgs/stuart_lk/","ORCSSite":"230"},
{"orcs":"9658","url":"https://bcparks.ca/parks/stuart-lake-marine/","oldUrl":"https://bcparks.ca/explore/parkpgs/stuart_lk/","ORCSSite":"9658"},
{"orcs":"8796","url":"https://bcparks.ca/parks/stuart-river/","oldUrl":"https://bcparks.ca/explore/parkpgs/stuart_rv/","ORCSSite":"8796"},
{"orcs":"9601","url":"https://bcparks.ca/parks/sue-channel/","oldUrl":"https://bcparks.ca/explore/parkpgs/sue_chan/","ORCSSite":"9601"},
{"orcs":"9815","url":"https://bcparks.ca/parks/sugarbowl-grizzly-den/","oldUrl":"https://bcparks.ca/explore/parkpgs/sugarbowl/","ORCSSite":"9815"},
{"orcs":"316","url":"https://bcparks.ca/parks/sukunka-falls/","oldUrl":"https://bcparks.ca/explore/parkpgs/sukunka/","ORCSSite":"316"},
{"orcs":"9540","url":"https://bcparks.ca/parks/sulphur-passage/","oldUrl":"https://bcparks.ca/explore/parkpgs/sulphur/","ORCSSite":"9540"},
{"orcs":"156","url":"https://bcparks.ca/parks/summit-lake/","oldUrl":"https://bcparks.ca/explore/parkpgs/summit_lk/","ORCSSite":"156"},
{"orcs":"204","url":"https://bcparks.ca/parks/sun-oka-beach/","oldUrl":"https://bcparks.ca/explore/parkpgs/sun_oka/","ORCSSite":"204"},
{"orcs":"9751","url":"https://bcparks.ca/parks/surge-narrows/","oldUrl":"https://bcparks.ca/explore/parkpgs/surge_nr/","ORCSSite":"9751"},
{"orcs":"9822","url":"https://bcparks.ca/parks/sustut/","oldUrl":"https://bcparks.ca/explore/parkpgs/sustut/","ORCSSite":"9822"},
{"orcs":"9777","url":"https://bcparks.ca/parks/sutherland-river/","oldUrl":"https://bcparks.ca/parks/sutherland-river/","ORCSSite":"9777"},
{"orcs":"16","url":"https://bcparks.ca/parks/swan-lake/","oldUrl":"https://bcparks.ca/explore/parkpgs/swan_lk/","ORCSSite":"16"},
{"orcs":"9077","url":"https://bcparks.ca/parks/swan-lake-kispiox-river/","oldUrl":"https://bcparks.ca/explore/parkpgs/swan_k_rv/","ORCSSite":"9077"},
{"orcs":"142","url":"https://bcparks.ca/parks/swiws/","oldUrl":"https://bcparks.ca/explore/parkpgs/swiws/","ORCSSite":"142"},
{"orcs":"73","url":"https://bcparks.ca/parks/sxwexwnitkw/","oldUrl":"https://bcparks.ca/explore/parkpgs/okana_fl/","ORCSSite":"73"},
{"orcs":"9495","url":"https://bcparks.ca/parks/sydney-inlet/","oldUrl":"https://bcparks.ca/explore/parkpgs/sydney/","ORCSSite":"9495"},
{"orcs":"202","url":"https://bcparks.ca/parks/syringa/","oldUrl":"https://bcparks.ca/explore/parkpgs/syringa/","ORCSSite":"202"},
{"orcs":"178","url":"https://bcparks.ca/parks/ta-chila/","oldUrl":"https://bcparks.ca/explore/parkpgs/tachila/","ORCSSite":"178"},
{"orcs":"9459","url":"https://bcparks.ca/parks/tahsish-kwois/","oldUrl":"https://bcparks.ca/explore/parkpgs/tahsish/","ORCSSite":"9459"},
{"orcs":"406","url":"https://bcparks.ca/parks/takla-lake-marine/","oldUrl":"https://bcparks.ca/explore/parkpgs/takla_lk/","ORCSSite":"406"},
{"
```

Comment:
We shouldn't be committing `^M` characters.

Reply:
just learned that, thanks